lib/hooks/update-data/place.js

```diff
@@ -0,0 +1,14 @@
+const { get, set } = require('lodash');
+
+module.exports = (settings, model) => {
+  const isAsru = get(model, 'meta.user.profile.asruUser');
+  const payload = get(model, 'meta.payload');
+
+  if (isAsru && payload.restrictions) {
+    const data = get(model, 'data');
+    set(data, 'data.restrictions', payload.restrictions);
+    return model.update(data);
+  }
```

Comment:
I don't think this makes it possible to remove all the restrictions? We might need to address that.

Possibly not in this PR though.